### PR TITLE
Move prefsCleaner.sh closer to POSIX

### PR DESCRIPTION
**Pull request by CelestialNebula**

### CODE REVIEW

File: prefsCleaner.sh [hunk 1]
```diff
@@ -1,114 +1,141 @@
...
+}
+
+fUsage_Help() {
+	printf "%s\n\n" "
+This script creates a backup of your prefs.js file before doing anything.
+It should be safe, but you can follow these steps if something goes wrong:
+
+1. Make sure Firefox is closed.
+2. Delete prefs.js in your profile folder.
+3. Delete Invalidprefs.js if you have one in the same folder.
```
Comment (CelestialNebula) [inline prefsCleaner.sh:48]: Is 'Invalidprefs.js' supposed to say 'invalid prefs.js'?

File: prefsCleaner.sh [hunk 2]
```diff
@@ -1,114 +1,141 @@
...
+	      [ -e lock ]                || \
+	      [ -e places.sqlite-wal ]
+	do
+		printf "\nThis Firefox profile seems to be in use.  Close Firefox and try again.\n" >&2
+		printf "Press Enter to continue."
+		read -r
 	done
 }
 
 fClean() {
```
Comment (CelestialNebula) [inline prefsCleaner.sh:75]: This was done to the best of my understanding of how the bash worked,
someone more familiar with bash should make sure this does an
equivalent.

**Comment by Thorin-Oakenpants** [comment]
Thanks. This is earthlng's to review. Hopefully he will pop in soon (he does at least once a month for release diffs)

**Comment by wael444** [comment]
?

**Pull request closed without merging**